tests/metrics/functional/test_classification.py

```diff
@@ -346,6 +347,52 @@ def test_auroc(pred, target, expected):
     assert score == expected
 
 
+def test_multiclass_auroc():
+    with pytest.raises(ValueError,
+                       match="Multiclass AUROC metric expects the target scores to be "
```

Comment:
we can catch just a part of the message, no need to have it complete, for example from the recommendation or i.e. section

Reply:
Done.